archive/cpio_old_le.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: cpio_old_le
+  title: cpio archive, old binary variant, little-endian
+  file-extension: cpio
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII

```

Comment:
This is kind of "fake usability", because it breaks whole format parsing when one encounters a non-ASCII character in an archive. Doing `pathName.toString(someEncoding)` is not totally hard, but at least it puts the liability on the user, not .ksy author(s).